sp800-63a/2_Validation.md

```diff
@@ -10,7 +10,7 @@ Identity validation is made up of two process steps: confirming the evidence is
 
 At IAL 2, agencies may allow applicants to present evidence digitally to facilitate remote proofing. Evidence presented in this manner will often be examined visually to determine if it appears fraudulent, counterfeit, or otherwise modified. The collection of evidence and validation of their authenticity may occur at the same time. For example, if an applicant provides a digital image of their driver's license, the system accepting that image may be able to automatically read the ID number or barcode and perform a lookup in the issuing state's motor vehicles department to confirm its authenticity.
 
-Agencies should also examine the evidence visually. Techniques for remote validation should be comparable to in-person validation. Table 5-2 in SP800-63A describe methods for validating identity evidence at each strength.
+Agencies should also examine the evidence visually. Techniques for remote validation should be comparable to in-person validation. Table 5-2 in SP800-63A describe methods for validating identity evidence. For remote validation, using offline data records such as those in the above example is an option. Cryptographic capabilities may also be helpful in performing identity validation remotely. It should be acknowledged that for IAL2 and IAL3, the ability to validate evidence remotely is limited.
```

Comment:
Suggest removing "for IAL2 and IAL3". This is true in general, and there is no requirement to validate evidence at IAL1.